snet_cli/arguments.py

```diff
@@ -249,6 +280,81 @@ def add_contract_options(parser):
         add_contract_function_options(contract_p, contract_name)
 
 
+def _add_service_publish_arguments(parser):
+    parser.add_argument("--no-register", action="store_true", help="does not register the published service")
+    parser.add_argument("--config", metavar="CONFIG_FILE", help="custom service.json file")
+    add_transaction_arguments(parser)
+
+
+def _add_service_update_arguments(parser):
+    parser.set_defaults(fn="update")
+    parser.add_argument("--new-price", help="new price to call the service", type=type_converter("uint256"))
+    parser.add_argument("--new-endpoint", metavar="ENDPOINT", help="new URL to call to reach the service")
```

Comment:
`ENDPOINT` metavar doesn't match the documentation

Reply:
Fixed